### PR TITLE
Android unittest parsing

### PR DESCRIPTION
**Pull request by paulaladdin**
Extended `linty_fresh` to support parsing of Android Buck Unittest files.

### CODE REVIEW

**Comment by paulaladdin** [comment]
👀 @apallin

**Comment by apallin** [comment]
I think that it may be best if we use the `passthrough` linter that Keith wrote and keep this logic in `clientci` to parse the unittest output.  I know we discussed this, but I think your original instinct to keep test parsing out of this considering its mostly "linting" may be the best option.

File: linty_fresh/main.py [hunk 1]
```diff
@@ -1,18 +1,11 @@
...
-from linty_fresh.linters import passthrough
-from linty_fresh.linters import pmd
-from linty_fresh.linters import pylint
-from linty_fresh.linters import swiftlint
-from linty_fresh.linters import xcodebuild
-from linty_fresh.reporters import github_reporter
-
 from typing import Any, Dict  # noqa
 
+from linty_fresh.linters import (android, buck_unittest, checkstyle, mypy,
```
Comment (apallin) [inline linty_fresh/main.py]: noice

**Comment by apallin** [comment]
So far so good, I think you still need to get it into the `main` function though as a `linter` option.

**Comment by apallin** [comment]
👍

**Pull request merged**